cmd/commands/ceph.go

```diff
@@ -33,9 +36,25 @@ var CephCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		logging.Info("running 'ceph' command with args: %v", args)
-		_, err := exec.RunCommandInOperatorPod(cmd.Context(), clientSets, cmd.Use, args, operatorNamespace, cephClusterNamespace, false)
-		if err != nil {
-			logging.Fatal(err)
+		if args[0] == "daemon" {
+			if len(args) > 2 {
```

Comment:
If there are not two args do they get an error?

Reply:
now it has
```bash
./bin/kubectl-rook-ceph ceph daemon
Warning: rook version 'rook: v1.14.0-alpha.0.80.gf84e932ec' is running a pre-release version of Rook.

Info: running 'ceph' command with args: [daemon]
Error: too few arguments to run the 'ceph daemon' command
```